app/src/main/java/com/example/hungrywolfes/ui/overview/OverviewFragment.kt

```diff
@@ -0,0 +1,80 @@
+package com.example.hungrywolfes.ui.overview
+
+import android.os.Bundle
+import android.util.Log
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.activityViewModels
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.RecyclerView
+import com.example.hungrywolfes.OverviewBinding
+import com.example.hungrywolfes.R
+import com.example.hungrywolfes.network.ListMealCategory
+import com.example.hungrywolfes.ui.CategoryAdapter
+import com.example.hungrywolfes.ui.ImagesAdapter
+
+
+class OverviewFragment : Fragment() {
+
+    private var binding: OverviewBinding? = null
+    private val viewModel: OverviewViewModel by activityViewModels()
```

Comment:
activityViewModels() is usually used to get the viewModel instance from the parent activity in order to communicate with other fragments. 
https://developer.android.com/topic/libraries/architecture/viewmodel#sharing

Here you could use **by viewModels()**.